core/src/test/java/nav/statusplattform/core/repositories/SampleData.java

```diff
@@ -248,63 +229,12 @@ public static MaintenanceEntity getRandomizedMaintenanceEntity() {
                 .setEnd_time(ZonedDateTime.now().plusDays(numberOfDays + 2));
     }
 
-    public static HelpTextEntity getRandomizedHelpTextEntity() {
-        return new HelpTextEntity()
-                .setNumber(getRandomFromLongArray(numbers))
-                .setType(getRandomServiceType())
-                .setContent(getRandomFromArray(helpTextDescriptions));
-    }
-
-    private static int getRandomFromLongArray(ArrayList<Integer> array) {
-        if (array.size() == 0) {
-            //Hit skal man ikke komme
-            return 0;
-        }
+    public static int randomPositiveInteger(int maxValue) {
         Random random = new Random();
-        return array.get(random.nextInt(array.size()));
-    }
-
-    public static List<HelpTextEntity> getHelpTextEntityWithServiceType(int length) {
-        List<HelpTextEntity> helpTexts = new ArrayList<>();
-        for (int i = 0; i < length; i++) {
-            helpTexts.add(new HelpTextEntity()
-                    .setNumber(i+1)
-                    .setType(ServiceType.TJENESTE)
-                    .setContent(getRandomFromArray(helpTextDescriptions)));
-        }
-        return helpTexts;
-    }
-
-    public static List<HelpTextEntity> getHelpTextEntityWithKomponentType(int length) {
-        List<HelpTextEntity> helpTexts = new ArrayList<>();
-        for (int i = 0; i < length; i++) {
-            helpTexts.add(new HelpTextEntity()
-                    .setNumber(i+1)
-                    .setType(ServiceType.KOMPONENT)
-                    .setContent(getRandomFromArray(helpTextDescriptions)));
+        int number = random.nextInt(maxValue);
+        if (number == 0) {
+            number = maxValue;
         }
```

Comment:
Why is this method never allowed to return 0?

Reply:
SampleData method :randomPositiveInteger() corrected to return 0 as well as positive integers.
randomPositiveInteger() corrected to manage errors from an input argument containing values that are zero or less.
SampleData method: added missing public from code to improve code uniformity.

Reply:
Next time, reply here _after_ the code (read: commits) reflecting this update has been pushed to the branch tracked by this PR =)

PS: And if so, also remember to "resolve" the conversation^^